donate/utility/acoustic/acoustic.py

```diff
@@ -106,7 +109,8 @@ def __init__(self, client_id, client_secret, refresh_token, server_number):
         self.api_xt_endpoint = self.api_xt_endpoint % server_number
         super().__init__(client_id, client_secret, refresh_token, server_number)
 
-    def _call_xt(self, xml):
+    def _call_xt(self, to, campaign_id, fields, bcc, save_to_db):
+        xml = transact_xml(to, campaign_id, fields, bcc, save_to_db)
```

Comment:
The scheduler really did not seem to like when I would try to send one "xml" object as a argument even after trying various methods, such as running transact_xml, the function which takes and formats the xml, and then passing it to the scheduler. 

So instead, I am now passing the required fields as arguments and then converting the xml in _call_xt.

Reply:
Follow up issue https://github.com/mozilla/donate-wagtail/issues/1655 created!